corehq/motech/repeaters/views/repeaters.py

```diff
@@ -141,6 +141,11 @@ def set_repeater_attr(self, repeater, cleaned_data):
         repeater.connection_settings_id = int(cleaned_data['connection_settings_id'])
         repeater.request_method = cleaned_data['request_method']
         repeater.format = cleaned_data['format']
+        name = cleaned_data.get('name')
+        if not name:
+            conn_settings = ConnectionSettings.objects.get(pk=repeater.connection_settings_id)
```

Comment:
Should this catch a DoesNotExist error?

Reply:
There's a `NOT NULL` constraint on the `connection_settings_id` field on the repeaters.


![Screenshot from 2023-01-18 10-25-19](https://user-images.githubusercontent.com/64970009/213120655-eece1aff-0f5f-450f-bbfe-07bc0f11fc53.png)